### DISCUSSION TITLE
How do i create a constant Line?

### DISCUSSION THREAD

**Discussion by alexnguyen98** [FormidableLabs/victory-native-xl#299] [General]
I am trying to create a constant line using this library. While i can add a Line component it is hard to determine the points array which would return an x start and ending in the children bounding area.

I want to create a constant line, with just declaring the y value (the height from the start).

**Comment by zibs** [comment]
I'm not sure I totally follow @alexnguyen98 -- can you show some examples/visuals of what you're expecting here?

**Reply by ZendomLtd** [reply]
Something like this, with just specifying the Y value without the x values
<img width="306" alt="Screenshot 2024-06-19 at 0 31 08" src="https://github.com/FormidableLabs/victory-native-xl/assets/172030918/7770bc5f-9348-482e-ab5d-f888c98d5fff">

**Reply by alexnguyen98** [reply]
@zibs @masiddee any news?

**Comment by masiddee** [comment]
@alexnguyen98 I'd like to help; could you further elaborate on exactly what it is you are attempting to achieve here?

**Comment by masiddee** [comment]
Moving this to Discussions

**Comment by zibs** [comment]
See #338

**Comment by G-Schumacher44** [comment]
## Extraction Plan for `cli.py` (#125)

**Proposed structure: `cli/` package**

1. `cli/hub.py` — hub start/stop/restart/status commands
2. `cli/memory.py` — memory service commands
3. `cli/remote.py` — remote MCP client management (start/stop/restart-remote)
4. `cli/stdio.py` — STDIO adapter setup and launchd config
5. `cli/launchd.py` — launchd plist rendering and management (~400 LOC, large enough to warrant its own module)
6. `cli.py` — entry point, CLI group, ctx.obj setup, global options only

**Key decisions:**
- Shared `ctx.obj["dry_run"]` and `MCPEnvironment` stay in `cli.py`
- No `ProcessRunner` abstraction — defer that to a future pass
- All existing entry points preserved unchanged

**Approved. Implementation on #125 can proceed.**
